src/__tests__/index.test.ts

```diff
@@ -1,24 +1,26 @@
-// const { persistCache } = jest.genMockFromModule('apollo-cache-persist');
-
-jest.mock('apollo-cache-persist', () => {
-	return { persistCache: jest.fn() };
-});
-
 import 'cross-fetch/polyfill';
 
+// const { persistCache } = jest.genMockFromModule('apollo-cache-persist');
+// import { ApolloClient } from 'apollo-client';
 import { BlueBase } from '@bluebase/core';
 import BlueBasePluginApollo from '@bluebase/plugin-apollo';
 import { CachePersistor } from 'apollo-cache-persist';
 import Plugin from '../index';
 
+jest.mock('apollo-cache-persist', () => {
+	return { persistCache: jest.fn() };
+});
+
+const client = jest.fn();
 jest.mock('apollo-cache-persist', () => ({ CachePersistor: jest.fn() }));
 
 test('Plugin should be correctly registered', async () => {
 	const BB = new BlueBase();
 
 	await BB.boot({ plugins: [BlueBasePluginApollo, Plugin] });
 	expect(BB.Plugins.has('@bluebase/plugin-apollo-cache-persist')).toBeTruthy();
+	BB.Filters.run('plugin.apollo.client', client);
```

Comment:
rather than doing this, add a filter on 'plugin.apollo.client'. And save a reference to that client from there. 

Then in the test, do client.clearStore. and check if purge was called as a result

Reply:
Sir i don't understand